WalletWasabi.Gui/MainWindow.xaml.cs

```diff
@@ -153,6 +153,7 @@ private async Task ClosingAsync()
 							Global.UiConfig.WindowState = WindowState;
 							Global.UiConfig.Width = Width;
 							Global.UiConfig.Height = Height;
+							Global.UiConfig.LastActiveTab = IoC.Get<IShell>().SelectedDocument?.Title;
```

Comment:
We need the type of the tab instead of the title for multiwallet compatibility. Introduce an enum for that.

Reply:
In fact, replace the `CoinListContainerType` with your enum, too, as it becomes redundant.

Reply:
Done.